list/style.go

```diff
@@ -67,7 +67,8 @@ func DefaultStyles() (s Styles) {
 		Foreground(lipgloss.AdaptiveColor{Light: "#A49FA5", Dark: "#777777"}).
 		Padding(0, 0, 1, 2) //nolint:mnd
 
-	s.StatusEmpty = lipgloss.NewStyle().Foreground(subduedColor)
+	s.StatusEmpty = lipgloss.NewStyle().Foreground(subduedColor).
+		SetString("No items.")
```

Comment:
Don't we need to do the same for `s.NoItems`?

Reply:
done!